cypress/e2e/room-directory/room-directory.spec.ts

```diff
@@ -72,32 +72,4 @@ describe("Room Directory", () => {
             expect(resp.chunk[0].room_id).to.equal(roomId);
         });
     });
-
-    it("should allow finding published rooms in directory", () => {
```

Comment:
Is this case covered in the spotlight tests?

Reply:
Finding public rooms is in spotlight's tests.

But sure, I could add a test that creates a private room, publishes it via the UI, and then tries to find it, just for completeness' sake.